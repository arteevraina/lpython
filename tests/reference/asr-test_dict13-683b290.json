{
    "basename": "asr-test_dict13-683b290",
    "cmd": "lpython --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/errors/test_dict13.py",
    "infile_hash": "ee757d46a81447cf45321822e379e83acbecd32578e300f27fc93d1b",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-test_dict13-683b290.stderr",
    "stderr_hash": "26039da2a42594ace7df2672a7bec3e88607caa7b45e35024b33df62",
    "returncode": 2
}