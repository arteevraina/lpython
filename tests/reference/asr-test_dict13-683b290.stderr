warning: The module 'lpython_builtin' located in $DIR/src/bin/../runtime/lpython_builtin.py cannot be loaded
 --> tests/errors/test_dict13.py:2:12
  |
2 |     print({complex(0.0, 1.0): 1.1}) # constant dict with complex value as key
  |            ^^^^^^^^^^^^^^^^^ imported here

semantic error: The symbol '_lpython_floordiv' not found in the module 'lpython_builtin'
   --> $DIR/src/bin/../runtime/lpython_builtin.py:209:15
    |
209 |     if (n_ - (n_ // 2)*2) == 0:
    |               ^^^^^^^ 
